config/docs.php

```diff
@@ -21,7 +21,7 @@
     |
     */
 
-    'header_title' => config('site.name', 'HydePHP').' Docs',
+    'header_title' => config('hyde.name', 'HydePHP').' Docs',
```

Comment:
Reference environment variable instead of configuration option in config/docs.php
```suggestion
    'header_title' => env('SITE_NAME', 'HydePHP').' Docs',
```

Reply:
Cherry picked to https://github.com/hydephp/develop/pull/970